resources/spec.yaml

```diff
@@ -12796,6 +13489,30 @@ definitions:
     type: string
     x-okta-tags:
       - Application
+  Org2OrgApplication:
+    properties:
+      settings:
+        $ref: '#/definitions/Org2OrgApplicationSettings'
+    x-okta-parent: '#/definitions/Application'
+    x-okta-tags:
+      - Application
+  Org2OrgApplicationSettings:
+    properties:
+      app:
+        $ref: '#/definitions/Org2OrgApplicationSettingsApp'
+    x-okta-parent: '#/definitions/ApplicationSettings'
+    x-okta-tags:
+      - Application
+  Org2OrgApplicationSettingsApp:
+    properties:
```

Comment:
Seems we should add a parent for it
`x-okta-parent: '#/definitions/ApplicationSettingsApplication'`

Reply:
fixed, thank you @VitaliiTytarenko-okta

Reply:
@VitaliiTytarenko-okta I'm not seeing the same pattern in `SamlApplicationSettings` to `SamlApplicationSettingsSignOn` where `SamlApplicationSettingsSignOn` would have a `x-okta-parent: '#/definitions/ApplicationSettingsApplication'`

https://github.com/okta/okta-management-openapi-spec/blob/mm_v2_10/resources/spec.yaml#L14701-L14776

Am I understanding this correctly? The suggestion is this?:

```
  Org2OrgApplicationSettingsApp:
    properties:
      acsUrl:
        type: string
      audRestriction:
        type: string
      baseUrl:
        type: string
    x-okta-parent: '#/definitions/ApplicationSettingsApplication'
    x-okta-tags:
      - Application
```

Reply:
Yes, this is correct.
Because 
```
  Org2OrgApplicationSettings:
    properties:
      app:
        $ref: '#/definitions/Org2OrgApplicationSettingsApp'
    x-okta-parent: '#/definitions/ApplicationSettings'
    x-okta-tags:
      - Application
.....
.....
  ApplicationSettings:
    properties:
      app:
        $ref: '#/definitions/ApplicationSettingsApplication'
```
That means that `app` method should return an instance of `ApplicationSettingsApplication`.
 Since `Org2OrgApplicationSettings` is the child of `ApplicationSettings`, to comply with the contract of `app` method `Org2OrgApplicationSettingsApp` should be the child of `ApplicationSettingsApplication`

Reply:
ok @VitaliiTytarenko-okta I made that change

Reply:
ok @VitaliiTytarenko-okta I made that change